compositor_pipeline/Cargo.toml

```diff
@@ -16,4 +16,5 @@ rtp = { workspace = true }
 webrtc-util = "0.8.0"
 socket2 = "0.5.5"
 rtcp = "0.10.0"
+rand = "0.8.5"
```

Comment:
I'm not sure what is considered a good practice in this case but `rand` is used in 2 crates so maybe consider making it a workspace dependency.